src/main/kotlin/io/fritz2/binding/seq.kt

```diff
@@ -1,54 +1,46 @@
 package io.fritz2.binding
 
 import kotlinx.coroutines.ExperimentalCoroutinesApi
-import kotlinx.coroutines.channels.ConflatedBroadcastChannel
-import kotlinx.coroutines.flow.Flow
-import kotlinx.coroutines.flow.asFlow
-import kotlinx.coroutines.flow.distinctUntilChanged
+import kotlinx.coroutines.FlowPreview
+import kotlinx.coroutines.flow.*
 import kotlinx.coroutines.flow.map
 
 data class Patch<out T>(val from: Int, val that: List<T>, val replaced: Int)
```

Comment:
Maybe a bit more comments here!?

Reply:
I just saw that a branched the new structure from this branch not the master (my fault), so there is no merging needed... ;-)